docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+    sonar:
+        image: sonarqube:community
+        ports:
+            - "9000:9000"
+        volumes:
+            - shared-volume:/mnt/data
+
+    visualization:
+        image: codecharta/codecharta-visualization
+        ports:
+            - "9001:80"
+
+    analysis:
+        image: codecharta/codecharta-analysis
+        volumes:
+            - shared-volume:/mnt/data
+
+volumes:
+    shared-volume: {}
```

Comment:
What is the path that is mounted by the volume `shared-volume`?

Reply:
shared-volume is an internal docker volume, currently without being mounted "from the outside". You can always copy files to and from it using docker cp. The volume is there so you can easily move files between the containers, like code files to and from sonar.